### PR TITLE
chore: added versioning api header

### PR DESCRIPTION
**Pull request by javiergarciavera**

### CODE REVIEW

File: test/SDK.test.ts [hunk 1]
```diff
@@ -85,6 +86,30 @@ describe('Sdk', () => {
     erc721Mocked.mockClear();
     signerMock.mockClear();
   });
+  describe('getApiVersion', () => {
+    it('should throw when transaction hash argument is not valid', async () => {
```
Comment (bergeron) [inline test/SDK.test.ts:90]: Is this test's description correct?  It looks unrelated to transaction hashes.

File: test/httpService.test.ts [hunk 2]
```diff
@@ -60,7 +61,7 @@ describe('httpService', () => {
       throw Error('test');
     });
 
-    const instance = new HttpService(BASEURL, APIKEY);
+    const instance = new HttpService(BASEURL, APIKEY, ApiVersion.V1);
```
[the remark this answers is not in the document]
Comment (javiergarciavera) [reply test/httpService.test.ts:64]: this could not be tested until we release API

[the remark this answers is not in the document]
Comment (javiergarciavera) [reply test/httpService.test.ts:64]: Fixed

**Review by sahar-fehri — APPROVED** [review]

**Comment by salimtb** [comment]
can you pls add a regex check on the api version + test , let's say for example i'm a user and i'll try to put `test` as version or any other inexisting version , i should expect the following error message:
`missing argument: Invalid api version. (location=\"[SDK.getOwnersByTokenAddress]\", code=MISSING_ARGUMENT, version=${version})`

**Comment by salimtb** [comment]
pls update the `usage.ts`  as well and include the use of the version , just to help users to understand how to use it

**Review by salimtb — APPROVED** [review]
